lldb/test/Shell/Reproducer/Swift/Inputs/Bridging.swift

```diff
@@ -1,5 +1,6 @@
 func main() {
-  let foo = Foo(i: 42)
+  Int x = 41 // Make sure we use the Swift Standard Library
```

Comment:
This C-style declaration doesn't really look like legal Swift?

Reply:
Btw, a reliable and obvious way of forcing the stdlib is to use `print`.

Reply:
You're totally right, Fred noticed the same thing on Friday. I'm positive I ran the tests but I was juggling different repos so I must've run the in the wrong build directory. 

This was fixed in https://github.com/apple/llvm-project/pull/1419